### PR TITLE
Feature/header footer

### PR DESCRIPTION
**Pull request by syncall**
First version of the Header/ Footer rework
Logos are still missing, but the links to the static pages are there

### CODE REVIEW

**Comment by sacdallago** [comment]
Thanks @syncall :) is there a version of this online that we could play around with or is the best idea to pull the branch and `npm start`?

Also: @pfent @kordianbruck I would say that we also review these sporadically to speed up the process a bit? :)

**Comment by syncall** [comment]
@sacdallago At the moment the only way is to pull and start the branch. Only the develop branch is sent to Docker.

File: client/Bundle.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  render () {
+    if (this.props.type && this.props.id && this.state.mod) {
+      return (<this.state.mod type={this.props.type} id={this.props.id} />)
+    }
+    return this.state.mod ? (<this.state.mod />) : (<div />)
+  }
+}
+
+Bundle.propTypes = {
+  load: React.PropTypes.func,
```
Comment (sacdallago) [inline client/Bundle.js:44]: Is it just me or the indent is a bit weird here?

**Comment by syncall** [comment]
@sacdallago Actually I think the Docker Setup is not quite right 😅 , you can have a look at it http://webpageg6.8ed630ce.svc.dockerapp.io/

**Review by sacdallago — APPROVED** [review]
As filler, this looks good to me. Content wise, we need to add text here and there, _everywhere_ 🎉 . Could you open up issues for the text sections you'd need to have input for?

E.g.:
- Logo proposals
- About page
- Social channels
- Legal (which as @kordianbruck suggested, you can take from https://got.show )

**Comment by sacdallago** [comment]
@syncall that might (is most likely) due to the docker node being bound to the old subscription, which is dead.

File: client/Bundle.js (hunk 1 above)
Comment (syncall) [reply client/Bundle.js:44]: Not sure what you mean ^^

**Comment by syncall** [comment]
@sacdallago but it is working. Dont't know why. It's probably more an Travis issue that the pull request gets pushed to docker

**Comment by kordianbruck** [comment]
Its a start. Can we merge? @syncall

**Comment by syncall** [comment]
Sure but there is still some stuff missing. Logo etc

Kordian Bruck <notifications@github.com> schrieb am Mi., 19. Apr. 2017,
22:41:

**Comment by sacdallago** [comment]
We can add that later! Please merge if at least there is a filler :)

**Comment by goldbergtatyana** [comment]
Very nice 🎉 

I love the footer. Not so sure about the header though. I imagine the header to look very similar to the one from https://got.show/. Also, I think we'd need to change the font of the logo in the upper left corner. Could you guys come up with a few suggestions for the font and posts screenshots here? 

Reviewing this kind of stuff is fuuuuun :)

**Comment by goldbergtatyana** [comment]
Just saw in #77 that you guys agreed to keep the search bar in the header. Fine for me. 

Still would go for a different font of the logo and the navigation buttons.

**Pull request merged**